.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+  - 2.2.0
```

Comment:
Ah, I meant adding multiple Rubies to test :wink: So we should at least test 2.1 and 2.2